crates/dead-man-switch-web/src/main.rs

```diff
@@ -359,6 +363,7 @@ async fn main() -> anyhow::Result<()> {
         .route("/dashboard", get(show_dashboard).post(handle_check_in))
         .route("/logout", post(handle_logout))
         .route("/timer", get(timer_data))
+        .route("/reset", get (handle_check_in))
```

Comment:
I think that `reset` it's really bad ambiguous term.
When I first looked at this PR I thought this was to stop the whole DMS.
That's what reset is for me.
Let's switch to
```suggestion
        .route("/check-in", get (handle_check_in))
```
or
```suggestion
        .route("/checkin", get (handle_check_in))
```

Reply:
Done